vali_objects/utils/elimination_manager.py

```diff
@@ -110,3 +112,52 @@ def _delete_eliminated_expired_miners(self):
                 except FileNotFoundError:
                     bt.logging.info(f"Zombie miner dir not found. Already deleted. [{miner_dir}]")
 
+    def _eliminate_MDD(self):
+        """
+        Checks the mdd of each miner and eliminates any miners that surpass MAX_TOTAL_DRAWDOWN
+        """
+        bt.logging.debug("checking for maximum drawdown.")
+        if self.shutdown_dict:
+            return
+
+        subtensor_weight_setter = SubtensorWeightSetter(
+            config=None,
+            wallet=None,
+            metagraph=None,
+            running_unit_tests=False
+        )
+
+        # Collect information from the disk and populate variables in memory
+        subtensor_weight_setter._refresh_eliminations_in_memory()
```

Comment:
eliminations are already fetched in this class. Can use self.eliminations for this